libs/@guardian/react-crossword/src/components/Cell.tsx

```diff
@@ -19,13 +23,14 @@ export const CellComponent = ({
 	guess = '',
 	theme,
 	isFocused,
-	isHighlighted,
+	isRelated,
+	isActive,
```

Comment:
Could we make this consistent with the clues? isHighlighted / isActive

Reply:
yeah 100% - i'm sure i've done that somewhere else, will do it here though